### PR TITLE
DCOS-13481: Feature (service form): allow GPUs for Mesos runtime

### PR DESCRIPTION
**Pull request by nLight**
This PR unblocks GPUs field for Mesos Runtime as it is the same thing as UCR and confirmed to support GPUs as well.

as GPUs field is still under the Container section we might want to reconsider its location in the form @leemunroe 

screen shot 2017-01-31 at 16 32 27

### CODE REVIEW

**Comment by leemunroe** [comment]
@nLight In this PR (https://github.com/dcos/dcos-ui/pull/1869) we're merging both advanced sections and moving the runtime options in to advanced. Wondering if this addresses your GPU location concerns?

File: plugins/services/src/js/components/forms/ContainerServiceFormSection.js [hunk 1]
```diff
@@ -151,11 +158,7 @@ class ContainerServiceFormSection extends Component {
   }
 
   getGPUSLabel() {
-    const {data, path} = this.props;
-    const typePath = this.getFieldPath(path, 'type');
-    const disabled = findNestedPropertyInObject(data, typePath) !== MESOS;
-
-    if (disabled) {
+    if (this.isGpusDisabled()) {
```
Comment (mlunoe) [inline plugins/services/src/js/components/forms/ContainerServiceFormSection.js]: Not that @jfurrow is working on something to make this easier: https://github.com/dcos/dcos-ui/pull/1867 (closed, but he will put up another PR).

This is not to block this PR, just to let you know 😉

**Comment by nLight** [comment]
@leemunroe you actually right, that solves the issue 🎉

**Comment by jfurrow** [comment]
🎉 :shipit:
